### PR TITLE
feat(misc): make createWorkspace quieter by default

### PR DESCRIPTION
**Pull request by AgentEnder**
## Current Behavior
Using the `createWorkspace` API triggers the same logs that a user gets with `create-nx-workspace`

## Expected Behavior
Folks using `createWorkspace` can provide their own logging provider.

image

Breaking down the above output: 
- 3 spinners that don't mention Nx
- `Fetching prettier...` - logged by the nx-plugin preset generator as @nrwl/js is initialized
- Output text logged by the nx-plugin plugin generator during preset generation 

## Related Issue(s)

Fixes #

### CODE REVIEW

**Review by xiongemi — APPROVED** [review]

File: packages/create-nx-workspace/src/create-workspace.ts [hunk 1]
```diff
@@ -79,16 +75,14 @@ export async function createWorkspace<T extends CreateWorkspaceOptions>(
...
-
   await recordStat({
     nxVersion,
     command: 'create-nx-workspace',
     useCloud: nxCloud,
     meta: messages.codeOfSelectedPromptMessage('nxCloudCreation'),
   });
+
+  return {
+    nxCloudInfo: nxCloudInstallRes?.stdout,
```
Comment (FrozenPandaz) [inline packages/create-nx-workspace/src/create-workspace.ts:86]: Let's also return the directory

File: packages/create-nx-workspace/src/create-workspace.ts [hunk 2]
```diff
@@ -79,16 +75,14 @@ export async function createWorkspace<T extends CreateWorkspaceOptions>(
...
-  if (nxCloud && nxCloudInstallRes?.code === 0) {
-    printNxCloudSuccessMessage(nxCloudInstallRes.stdout);
-  }
-
   await recordStat({
     nxVersion,
     command: 'create-nx-workspace',
     useCloud: nxCloud,
     meta: messages.codeOfSelectedPromptMessage('nxCloudCreation'),
   });
```
Comment (FrozenPandaz) [inline packages/create-nx-workspace/src/create-workspace.ts:83]: Move this out of here for now. Let's not collect stats on this.

I do think it is valuable to collect this information.. but we need to make sure other CLI authors do not do this by default.

File: packages/create-nx-workspace/src/utils/error-utils.ts [hunk 3]
```diff
@@ -1,10 +1,23 @@
...
+      bodyLines?: string[];
+    }
   ) {
-    super(logMessage);
+    super(message);
+    this.code = opts?.code;
+    this.logFile = opts?.logFile;
+    this.title = opts?.title;
+    this.bodyLines = opts?.bodyLines;
     this.name = 'CreateNxWorkspaceError';
```
Comment (FrozenPandaz) [inline packages/create-nx-workspace/src/utils/error-utils.ts:21]: I think this can be changed back.

**Review by FrozenPandaz — CHANGES_REQUESTED** [review]

**Review by FrozenPandaz — APPROVED** [review]

File: packages/create-nx-workspace/src/create-preset.ts [hunk 4]
```diff
@@ -42,7 +42,7 @@ export async function createPreset<T extends CreateWorkspaceOptions>(
   if (process.env.NX_VERBOSE_LOGGING !== 'true') {
     args = '--quiet ' + args;
   }
-  const command = `g ${preset}:preset ${args}`;
+  const command = `g ${preset}:preset ${args} --quiet`;
```
Comment (FrozenPandaz) [inline packages/create-nx-workspace/src/create-preset.ts]: It's on the line above.

**Review by FrozenPandaz — APPROVED** [review]

**Pull request merged**